pkg/mcp/loader.go

```diff
@@ -230,10 +230,6 @@ func (sm *SessionManager) closeClients(id string) {
 // RestartServerDeployment restarts the server in the currently used backend, if the backend supports it.
 // If the backend does not support restarts, then an [ErrNotSupportedByBackend] error is returned.
 func (sm *SessionManager) RestartServerDeployment(ctx context.Context, server ServerConfig) error {
-	if server.Command == "" {
```

Comment:
I think the original intent of this is to not call the below function for remote MCP servers.

Can we add this back, but change the check to `server.Runtime == types.RuntimeRemote`.

Reply:
OK, I can add it here.